packages/cli/src/agent.ts

```diff
@@ -197,24 +198,37 @@ agentUpgradeCommand
     'An optional FHIR search criteria to resolve the agent(s) to upgrade. Mutually exclusive with [agentIds...] arg'
   )
   .option(
-    '--version <version>',
-    'An optional version to upgrade to. Defaults to the latest version if flag not included'
+    '--agentVersion <version>',
+    'An optional agent version to upgrade to. Defaults to the latest version if flag not included'
+  )
+  .option(
+    '--force',
+    'Used to force an upgrade in a scenario where a pending upgrade may have gotten into a bad state. USE AT YOUR OWN RISK!'
```

Comment:
Dialing down the temperature 😄 

```suggestion
    'Forces an upgrade when a pending upgrade is in an inconsistent state. Use with caution.'
```